### ISSUE TITLE
IRC logs not colored anymore since v2.0.0

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments#783]
(Original issue 1076 created by Kwpolska on 2015-01-20T16:59:54.314850+00:00)

Pygments v1.6.0:

```html
<div class="highlight"><pre><span class="cp">2015-01-20 15:42:12 </span><span class="nt">&lt;ChrisWarrick&gt; </span><span class="na">KwBot:</span> hi
</pre></div>
```

Pygments v2.0.0–v2.0.2:

```html
<div class="highlight"><pre>2015-01-20 15:42:12 &lt;ChrisWarrick&gt; KwBot: hi
</pre></div>
```

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2015-01-21T06:16:58.122506+00:00)

How are you generating this snippet? Is the lexer autodetected, or are you specifying "irclogs" explicitly?

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2015-01-21T06:43:32.904011+00:00)

Never mind, got it.

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2015-01-21T06:43:48.526644+00:00)

Fix highlighting of certain IRC logs formats (closes #1076).

→ <<cset b27cb08e6a40>>

**Comment by Anteru** [comment]
(Original comment by Kwpolska on 2015-01-21T08:58:41.697939+00:00)

Thanks for the quick fix!  (for the record, specifying the IRC lexer directly.)
